presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -10,29 +10,392 @@ Config Properties
 
 These configuration options may require tuning in specific situations:
```

Comment:
I think it's very good idea, but If you don't mind I would wait with that until we collect all descriptions we want - that way it will be easier to split them I think.